whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/InetnumStatusNotSetSpec.groovy

```diff
@@ -0,0 +1,284 @@
+package net.ripe.db.whois.spec.update
+
+import net.ripe.db.whois.common.IntegrationTest
+import net.ripe.db.whois.spec.BaseQueryUpdateSpec
+import net.ripe.db.whois.spec.domain.AckResponse
+import net.ripe.db.whois.spec.domain.Message
+import spock.lang.Ignore
+
+@org.junit.experimental.categories.Category(IntegrationTest.class)
+class InetnumStatusNotSetSpec extends BaseQueryUpdateSpec {
+
+    // in the database the only parent that exists for NOT-SET resources is ALLOCATED UNSPECIFIED
+    // and the only child resource status with a NOT-SET resource as a parent is LEGACY
+    @Override
+    Map<String, String> getTransients() {
+        [
+                "ALLOC-UNS": """\
+                inetnum:      192.168.0.0 - 192.169.255.255
+                netname:      TEST-NET-NAME
+                descr:        TEST network
+                country:      NL
+                org:          ORG-HR1-TEST
+                admin-c:      TP1-TEST
+                tech-c:       TP1-TEST
+                status:       ALLOCATED UNSPECIFIED
+                mnt-by:       RIPE-NCC-HM-MNT
+                mnt-lower:    LIR-MNT
+                source:       TEST
+                """,
+                "LEGACY-CHILD": """\
+                inetnum:    192.168.1.0 - 192.168.1.50
+                netname:    TEST-NETNAME
+                country:    NL
+                admin-c:    TP1-TEST
+                tech-c:     TP1-TEST
+                status:     LEGACY
+                mnt-by:     LIR-MNT
+                source:     TEST
+                """,
+                "NOTSET": """\
+                inetnum:    192.168.1.0 - 192.168.1.255
+                netname:    TEST-NETNAME
+                country:    NL
+                admin-c:    TP1-TEST
+                tech-c:     TP1-TEST
+                status:     NOT-SET
+                mnt-by:     LIR-MNT
+                source:     TEST
+                """,
+                "NOTSET-200": """\
+                inetnum:    192.168.200.0 - 192.168.200.50
+                netname:    TEST-NETNAME
+                country:    NL
+                admin-c:    TP1-TEST
+                tech-c:     TP1-TEST
+                status:     NOT-SET
+                mnt-by:     LIR-MNT
+                source:     TEST
+                """
+        ]}
+
+    //TODO: ALLOCATED should NOT be allowed but currently is
+    def "change NOT-SET status to allowed statuses with ALLOCATED UNSPECIFIED parent"() {
+        given:
+        syncUpdate(getTransient("ALLOC-UNS") + "password: owner3\npassword: hm\npassword: lir")
+        syncUpdate(getTransient("NOTSET") + "override: denis,override1")
+
```

Comment:
Why adding data with a specific password, if you use for the other the override?
Just using override for all could simplify code (that part - the initial data insertion - 
is not really part of the test, I would say)